src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -54,6 +54,14 @@
     <AssemblyOriginatorKeyFile>..\Common\PublicKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.AI.PerfCounterCollector, Version=2.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Microsoft.ApplicationInsights.PerfCounterCollector.2.2.0\lib\net45\Microsoft.AI.PerfCounterCollector.dll</HintPath>
```

Comment:
UnobservedTaskException can happen when you forgot to add `.ContinueWith`. So it may be useful. You can postpone it to the next version